var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py

```diff
@@ -66,8 +72,10 @@ class PyPennylaneLightningKokkos(CMakePackage, PythonExtension, CudaPackage, ROC
     depends_on("py-pybind11", type="build")
```

Comment:
Seems like this should actually be a link dep?

Reply:
Pybind11 is header-only, so any artifacts generated at build time should have everything needed to fully support import from Python. Should be fine to leave as "build" only here.

Reply:
In Spack, "build" dep means add `<prefix>/bin` to the PATH, while "link" dep means inject `-L<prefix>/lib` and `-I<prefix>/include` via the compiler wrappers. So it sounds like link would make more sense here.

Reply:
Ah, I misunderstood, thanks. I'll update this across the board. Thanks.